### PR TITLE
Use spies for console output in tests

### PR DESCRIPTION
**Pull request by bangbay-bluetiger**
### Description

- Use spies for tests that intentionally output console messages

### Related ticket(s)

CMDCT-3825

---
### How to test

1. cd to `macpro-mdct-mfp/services/app-api`
2. Run `yarn coverage`
3. Verify tests pass and coverage is unaffected
4. Verify console messages are gone

### Pre-review checklist

- [x] I have performed a self-review of my code

### CODE REVIEW

**Review by britt-mo — APPROVED** [review]
Verified that test coverage is unchanged and there are no more console errors

File: services/app-api/handlers/banners/create.test.ts [hunk 1]
```diff
@@ -21,9 +21,18 @@ const testEvent: APIGatewayProxyEvent = {
...
 };
 
+const consoleSpy: {
+  debug: jest.SpyInstance<void>;
+  error: jest.SpyInstance<void>;
+} = {
+  debug: jest.spyOn(console, "debug").mockImplementation(),
+  error: jest.spyOn(console, "error").mockImplementation(),
+};
+
```
Comment (gmrabian) [inline services/app-api/handlers/banners/create.test.ts:31]: is there any way we could make this shared in the setupJest file?

**Review by gmrabian — APPROVED** [review]
terminal output is clean :)

File: services/app-api/handlers/banners/create.test.ts (hunk 1 above)
Comment (bangbay-bluetiger) [reply services/app-api/handlers/banners/create.test.ts:31]: Possibly. I looked into it a little bit, but didn't find a ready answer. Open to refactoring this in another cleanup ticket.

Comment (bangbay-bluetiger) [reply services/app-api/handlers/banners/create.test.ts:31]: One reason I wouldn't want to set this up globally is I would want to catch console messages we aren't expecting.

**Pull request merged**